content/en/continuous_integration/guides/use_ci_jobs_failure_analysis.md

```diff
@@ -0,0 +1,144 @@
+| script         | Failures caused by syntactic errors in the script in the CI Job.                                    | `No tests ran due to file or directory not found.`                                                                                                    |
+
+{{% /tab %}}
+{{% tab "unknown" %}}
+
+| Subdomain | Description                     | Example                                                 |
+|-----------|---------------------------------|---------------------------------------------------------|
+| unknown   | Error could not be categorized. | `Job failed with exit code 1. View full logs or trace.` |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Supported CI providers
+
+CI jobs failure analysis is available for the following CI providers:
+
+* [GitHub Actions][1]
+* [GitLab][2]
+
+Notice that CI Job Logs collection enabled is required and the logs need to be indexed. If you need to setup CI Job Logs collection, check out the setup instructions for your CI provider on the [Pipeline Visibility page][6].
+
+<div class="alert alert-info">If you are interested in CI jobs failure analysis but your CI provider is not supported yet, fill out <a href="https://forms.gle/vSrqS5QwitgHf9wG6" target="_blank">this form</a>.</div>
+
+## Identify the most recurrent errors in your CI pipelines
+
+### Using CI Health page
+
+You can access to the [CI Health page][3] to see a breakdown of the errors in your CI pipelines split by error domain.
+
+- Go to [CI Health][3]
+- Click on a CI pipeline.
+- Check the `Breakdown` column in the `Failed Executions` section.
+
+{{< img src="continuous_integration/ci_health_failed_executions_breakdown.png" alt="CI Job Failure analysis breakdown in CI Health" width="90%">}}
+
+[CI Health][3] provides a high-level overview of the health and performance of your CI pipelines. It helps DevOps and engineering teams monitor CI jobs, detect failures, and optimize build performance.
+
+### Using facets
+
+You can use the facets `@error.message`, `@error.domain`, and `@error.subdomain` to identify the most recurrent errors in your CI pipelines. Using those facets, you can create custom dashboards and notebooks for your needs.
+
+{{< img src="continuous_integration/failed_jobs_ai_gen_errors_facets.png" alt="Failed CI Jobs filtered by error.domain and error.subdomain" width="90%">}}
+
+Notice that these facets are only available using the `ci_level:job` in your queries. In case that the CI jobs failures analysis cannot be computed (e.g., you are not using a supported CI provider), these facets will contain the error information coming from the CI provider.
+
+### Using the dashboard template
+
+You can also import the CI Visibility - CI Jobs Failure Analysis dashboard template:
+
+- Open the [civisibility-ci-jobs-failure-analysis-dashboard.json][4] dashboard template and copy the content in the clipboard.
+- Create a [New Dashboard][5] in Datadog.
+- Paste the copied content in the new dashboard.
+- Save the dashboard.
```

Comment:
For the New Dashboard link, it'd be great if we could deep link directly into the screen in Datadog where the user creates dashboards. I don't know if that's possible, though.
```suggestion
1. Open the [civisibility-ci-jobs-failure-analysis-dashboard.json][4] dashboard template and copy the contents into the clipboard.
2. Create a [New Dashboard][5] in Datadog.
3. Paste the copied content into the new dashboard.
4. Save the dashboard.
```